config/routes.rb

```diff
@@ -5,4 +5,5 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/users/:id", to: "users#index"
```

Comment:
Do we want to handroll routes like this or do we want to use resources and namespaces?